Bigtable/tests/Snippet/DataClientTest.php

```diff
@@ -226,6 +233,128 @@ public function testReadRow()
         );
     }
 
+    public function testReadModifyWriteRowAppend()
+    {
+        $readModifyWriteRowResponse = (new ReadModifyWriteRowResponse)
+            ->setRow(
+                (new Row)
+                    ->setFamilies(
+                        [
```

Comment:
We can put the opening bracket next to the parenthesis, `setFamilies([...])`. It should help keep the spacing a little more readable.